#!/bin/sh -e

# All lines beginning with `# DP:' are a description of the patch.
# DP: Calls x-terminal-emulator instead of xterm

if [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
    -patch) patch -f --no-backup-if-mismatch -p0 < $0;;
    -unpatch) patch -f --no-backup-if-mismatch -R -p0 < $0;;
    *)
	echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
	exit 1
esac

exit 0

diff -u src/actions.c src/actions.c
--- src/actions.c	Sat Dec 16 16:07:05 2000
+++ src/actions.c	Sat Dec 16 18:15:07 2000
@@ -9,7 +9,7 @@
 
 /* Initialization of the key structure */
 rp_action key_actions[] = { {KEY_PREFIX, 0, 0, generate_prefix},
-                            {XK_c, -1, "xterm", spawn},
+                            {XK_c, -1, "x-terminal-emulator", spawn},
 			    {XK_e, -1, "emacs", spawn},
 			    {XK_p, -1, 0, prev_window},
 			    {XK_n, -1, 0, next_window},
